### PR TITLE
fix: ENGAGEMENT CENTER Improve the display of program card - MEED-707 - Meeds-io/MIPs#13

### PR DESCRIPTION
**Pull request by MayTekayaa**
Prior to this change, the program card contains only the program cover image, title and points per user.
This change will adjust the display for more visibility by adding the owners program part.

### CODE REVIEW

File: portlets/src/main/resources/locale/portlet/Challenges_en.properties [hunk 1]
```diff
@@ -95,7 +95,7 @@ challenges.domainDeleteOrDisabled=Challenge program is disabled or deleted
 programs.button.addProgram=Add a program
 programs.button.editProgram=Edit program
 programs.cover.default=Program default cover
-programs.budget=about {0} per user
+programs.budget=About {0} per user
```
Comment (AzmiTouil) [inline portlets/src/main/resources/locale/portlet/Challenges_en.properties]: should be done in crowdin

File: portlets/src/main/resources/locale/portlet/Challenges_en.properties [hunk 2]
```diff
@@ -116,7 +116,8 @@ programs.cover.uploadErrorMessage=Error when uploading image
 programs.details.label.description=Description
 programs.details.label.rulesOfProgram=Rules of program
 programs.details.label.perUser=per user
-programs.details.label.points=points
+programs.details.label.points=Points
```
Comment (AzmiTouil) [inline portlets/src/main/resources/locale/portlet/Challenges_en.properties]: should be done in crowdin after merge

File: portlets/src/main/webapp/skin/less/challenges.less [hunk 3]
```diff
@@ -134,6 +134,42 @@
...
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-weight: bold;
+  }
```
Comment (AzmiTouil) [inline portlets/src/main/webapp/skin/less/challenges.less]: Avoid adding news custom classes please!

File: portlets/src/main/webapp/vue-app/engagement-center/components/common/AvatarsList.vue [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    },
+  },
+  data() {
+    return {
+      maxAvatarsToShow: 4,
+    };
+  },
+  computed: {
+    avatarsCount () {
+      return this.avatars.length;
```
Comment (AzmiTouil) [inline portlets/src/main/webapp/vue-app/engagement-center/components/common/AvatarsList.vue]:
```suggestion
      return this.avatars?.length;
```

**Review by AzmiTouil — CHANGES_REQUESTED** [review]

File: portlets/src/main/webapp/vue-app/engagement-center/components/common/AvatarsList.vue (hunk 4 above)
Comment (MayTekayaa) [reply portlets/src/main/webapp/vue-app/engagement-center/components/common/AvatarsList.vue]: done!

File: portlets/src/main/webapp/skin/less/challenges.less (hunk 3 above)
Comment (MayTekayaa) [reply portlets/src/main/webapp/skin/less/challenges.less]: done!

File: portlets/src/main/resources/locale/portlet/Challenges_en.properties (hunk 2 above)
Comment (MayTekayaa) [reply portlets/src/main/resources/locale/portlet/Challenges_en.properties]: done!

File: portlets/src/main/resources/locale/portlet/Challenges_en.properties (hunk 1 above)
Comment (MayTekayaa) [reply portlets/src/main/resources/locale/portlet/Challenges_en.properties]: done!

File: portlets/src/main/webapp/vue-app/engagement-center/components/programs/ProgramOwnersDrawer.vue [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    </exo-drawer>
+  </v-app>
+</template>
+
+<script>
+
+export default {
+  data() {
+    return {
+      displayBackIcon: false,
```
Comment (AzmiTouil) [inline portlets/src/main/webapp/vue-app/engagement-center/components/programs/ProgramOwnersDrawer.vue]: useless!

File: portlets/src/main/webapp/vue-app/engagement-center/components/programs/ProgramOwnersDrawer.vue [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+  },
+  created() {
+    this.$root.$on('open-drawer', this.open);
+  },
+  methods: {
+    close() {
+      this.$refs.ownersDetails.close();
+    },
+    open(avatars, displayBackIcon) {
+      this.displayBackIcon = displayBackIcon;
```
Comment (AzmiTouil) [inline portlets/src/main/webapp/vue-app/engagement-center/components/programs/ProgramOwnersDrawer.vue]:
```suggestion
```

File: portlets/src/main/webapp/vue-app/engagement-center/components/programs/ProgramCard.vue [hunk 7]
```diff
@@ -68,16 +31,61 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
...
+    </div>
+    
+    <div class="d-flex mt-2 mx-2">
+      <div class="pa-1">
+        <span class="my-auto caption text-light-color"> {{ this.$t('programs.details.label.hosts') }} </span>
       </div>
+      <v-spacer />
+      <engagement-center-avatars-list
+        :avatars="owners"
+        :program="program"
```
Comment (AzmiTouil) [inline portlets/src/main/webapp/vue-app/engagement-center/components/programs/ProgramCard.vue]:
```suggestion
```

File: portlets/src/main/webapp/vue-app/engagement-center/components/common/AvatarsList.vue [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+    seeMoreAvatarsToDisplay () {
+      return this.avatarsCount >= this.maxAvatarsToShow && this.avatars[this.maxAvatarsToShow - 1] || null;
+    },
+    showMoreAvatarsNumber() {
+      return this.avatarsCount - this.maxAvatarsToShow + 1;
+    },
+  },
+  methods: {
+    openDrawer() {
+      this.$root.$emit('open-drawer', this.avatars);
```
Comment (AzmiTouil) [inline portlets/src/main/webapp/vue-app/engagement-center/components/common/AvatarsList.vue]:
```suggestion
      this.$root.$emit('open-avatars-drawer', this.avatars);
```

**Review by AzmiTouil — CHANGES_REQUESTED** [review]

File: portlets/src/main/webapp/vue-app/engagement-center/components/common/AvatarsList.vue (hunk 8 above)
Comment (MayTekayaa) [reply portlets/src/main/webapp/vue-app/engagement-center/components/common/AvatarsList.vue]: modified!

File: portlets/src/main/webapp/vue-app/engagement-center/components/programs/ProgramCard.vue (hunk 7 above)
Comment (MayTekayaa) [reply portlets/src/main/webapp/vue-app/engagement-center/components/programs/ProgramCard.vue]: removed!

File: portlets/src/main/webapp/vue-app/engagement-center/components/programs/ProgramOwnersDrawer.vue (hunk 6 above)
Comment (MayTekayaa) [reply portlets/src/main/webapp/vue-app/engagement-center/components/programs/ProgramOwnersDrawer.vue]: removed!

**Review by AzmiTouil — APPROVED** [review]

**Pull request merged**